<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>922</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1171</width>
      <height>1071</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabQuery">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableWidget" name="tableResult">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>721</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExec">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>530</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textSQL">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>530</y>
        <width>721</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnChangeDB">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>60</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Change DB</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>68</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Results</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>SQL Query</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
